php7/README

```diff
@@ -0,0 +1,22 @@
+================================================================================
+  PHP 7.3 yacc grammar
+================================================================================
+
+This is an adaption of PHP's 7.3 Yacc grammar for lrpar. There are few things
+that are currently not supported. First, the dangling else problem in PHP is
```

Comment:
I suggest a paragraph break before "first". Also, in a sense, this isn't something that doesn't work. It does work, but it produces a warning.

Maybe the README structure should be:

```
This is an adaption of PHP's 7.3 Yacc grammar / lexer for grmtools.

There are two limitations to be aware of:

1) ...
2) ...

You may also notice shift/reduce and reduce/reduce errors in the grammar.
These are because of [talk about the T_NOELSE thing].
```